From 9f23e44cbeb6555d5a0afd9d624e515e661a9c19 Mon Sep 17 00:00:00 2001
From: Clayton Errington <cerringtondesigns@me.com>
Date: Mon, 23 Feb 2015 15:36:40 -0600
Subject: [PATCH 080/214] improve the quality of anaytics

Improve the quality of Google Analytics as found in:
http://veithen.github.io/2015/01/05/jekyll-improving-ga-data-quality.html

Reduce and eliminate referrer spam in Google Analytics as found in:
http://veithen.github.io/2015/01/21/referrer-spam.htmlimprove the quality of analytics http://veithen.github.io/2015/01/05/jekyll-improving-ga-data-quality.html
---
 _includes/analytics.html | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/_includes/analytics.html b/_includes/analytics.html
index e70d8bf..4431883 100644
--- a/_includes/analytics.html
+++ b/_includes/analytics.html
@@ -5,9 +5,12 @@
 		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
 		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
 		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
+		
 		ga('create', '{{ site.google_analytics }}', 'auto');
-		ga('send', 'pageview');
+		ga('send', 'pageview', {
+		  'page': '{{ page.url }}',
+		  'title': '{{ page.title | replace: "'", "\\'" }}'
+		});
 	</script>
 	<!-- End Google Analytics -->
 {% endif %}
-- 
2.17.1

